src/components/TestSuiteRun/TestSuiteRunSummary.jsx

```diff
@@ -35,11 +36,29 @@ import {
 } from "@material-ui/icons";
 import Test from "../../tests/Test";
 
-import { setCurrentTestRun } from "../../actions/testSuiteRunActions";
+import { setCurrentTestRun as setCurrentTestRunAction } from "../../actions/testSuiteRunActions";
 
 import "./TestSuiteRunSummary.css";
 
 class TestSuiteRunSummaryBase extends React.Component {
+  static propTypes = {
+    keystore: PropTypes.shape({
+      note: PropTypes.string,
+      type: PropTypes.string,
+      version: PropTypes.string,
+    }).isRequired,
+    testSuiteRun: PropTypes.shape({
+      currentTestRunIndex: PropTypes.number,
+      started: PropTypes.bool,
+      testRuns: PropTypes.arrayOf(PropTypes.shape({})),
+    }).isRequired,
+    setCurrentTestRun: PropTypes.func,
+  };
+
+  static defaultProps = {
+    setCurrentTestRun() {},
```

Comment:
I don't think we want to be setting a default here. This should be passed in from the redux connection function and so should be required in the prop checker. This would then throw a warning because the labeling is off. (see comment below)

Reply:
Updated.